static/js/components/SearchTextbox_test.js

```diff
@@ -17,7 +17,8 @@ describe("SearchTextbox", () => {
   })
 
   const render = (props = {}) =>
-    shallow(
+    // mount is necessary to check focus
+    mount(
```

Comment:
might be worth keeping the `shallow` based test renderer around for the tests that don't need mount - what do you think? you can quickly make a shallow renderer using the `configureShallowRenderer` test util btw.

Reply:
There's only a few of them, but I reverted `render()` to use shallow as before and added a `renderMount` method for the tests that check on focus.